SU-HSDO/suhumsci
# ARCH-168 removed heading dash from sidebar menus

# READY FOR REVIEW

# Summary
- Removed heading dash below sidebar menu title

# Needed By (Date)
- End of sprint (7/19)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that menu title on subpages does not have a heading dash below it

# Affected Projects or Products
- Archeaology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-168
- BUGHERD-22

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
